start.sh

```diff
@@ -63,6 +63,21 @@ https://mail.{\$NC_DOMAIN}:443 {
 CADDY
 fi
 
+if [ -n "$(dig A +short nextcloud-aio-jellyfin)" ] && ! grep -q nextcloud-aio-jellyfin /Caddyfile; then
+    cat << CADDY >> /Caddyfile
+https://jellyfin.{\$NC_DOMAIN}:443 {
```

Comment:
Maybe media.yourdomain.com looks nicer?
```suggestion
https://media.{\$NC_DOMAIN}:443 {
```

Reply:
Of course, I just wanted to stay specific to jellyfin, since there is also a container for plex. But in principle the 2 shouldn't be used together 👍 but media is nicer